### DISCUSSION TITLE
Add "fields" parameter to /fields endpoint

### DISCUSSION THREAD

**Discussion by JUVOJustin** [directus/directus#5929] [Feature Requests]
I am currently working on an app where the user can choose which items to query for. For example dynamically building the filters etc. . It is already possible to get all fields for a collection, but in case of building filters it would be great to get the related fields to. 

The way the "fields" parameter works on items would be perfect.

All projects with auto populating fields might profit from this.

**Comment by rijkvanzanten** [comment]
The end-goal is to have the same query parameters across the board 🙂 

The params for fields/collections/relations are slightly more complex, as they have to be run _after_ the data is retrieved from the DB instead of it them being part of the executed SQL query

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
